src/content/packages-and-plugins/using-packages.md

```diff
@@ -124,10 +124,10 @@ To add the package, `css_colors`, to an app:
 
 ### Adding a package dependency to an app using `flutter pub add`
```

Comment:
I know unrelated to this PR, but consider moving this section before the section manually adding the dependency.

Using the commands help ensure developers get the latest versions of dependencies and is less prone to mistakes.